content/en/platform/corda/5.0-dev-preview-1/packaging/cordapp-builder.md

```diff
@@ -64,7 +64,7 @@ If a previous installation of CorDapp Builder CLI exists, remove it. See [deleti
 
 ### Automatic installation
 
-1. Download the universal installer from [Artifactory](https://software.r3.com/artifactory/corda-os-maven-stable/net/corda/cordapp-builder/%5BRELEASE%5D/cordapp-builder-%5BRELEASE%5D-installer.jar).
+1. Download the [universal installer](https://download.corda.net/cordapp-builder/5.0.0-DevPreview-1.0/cordapp-builder-installer.jar).
 2. Run the following command.
     ```bash
     java -jar cordapp-builder-installer-*.jar
```

Comment:
this is now simply called `cordapp-builder-installer.jar` while the current command would fail as there's no trailing dash after the `installer` suffix